### DISCUSSION TITLE
Error in the data folder

### DISCUSSION THREAD

**Discussion by mgutierreztudelft** [tudelft/AE4353-Y24#25] [General]
Hello, last day I could access to the data folder correctly. However, now I do not have access and I have not touched anything relevant. I attach a picture where it shows that I am putting the path correctly and despite that, in the "data" folder at the left it does not appear.

![image](https://github.com/user-attachments/assets/e045b439-40ca-48d9-9ca4-a478c05da697)

Any idea of what could be happening here? without the data I am not able to continue the exercises.
Thanks

**Comment by yilun-wu** [comment]
The mount line seems fine with me. Can you check if there are files under `/home/mgutierrezfern/Downloads/ae4353_dataset`?
